adcm_aio_client/core/objects/_base.py

```diff
@@ -84,3 +90,24 @@ def __init__(self: Self, parent: Parent, requester: Requester, data: dict[str, A
 
     def get_own_path(self: Self) -> Endpoint:
         return *self._parent.get_own_path(), self.PATH_PREFIX, self.id
+
+
+class MaintenanceMode:
+    def __init__(
+        self: Self, maintenance_mode_status: MaintenanceModeStatus, requester: Requester, path: Endpoint
+    ) -> None:
+        self.maintenance_mode_status = maintenance_mode_status
```

Comment:
make protected, it should be available to be changed publicly 

you may add propery like `value`, so user can access it

Reply:
done